lib/common.js

```diff
@@ -55,7 +55,7 @@ const isUtf8Representable = function(buffer) {
 }
 
 //  Array where all information about all the overridden requests are held.
-let requestOverride = []
+let requestOverrides = {}
```

Comment:
🙈

Reply:
wait, what? How did this ever work?

Reply:
In JavaScript any object can be used as a dictionary, and I guess `_.keys()` did the right thing…